src/Microsoft.AspNetCore.Razor.Language/DefaultRazorTagHelperBinderPhase.cs

```diff
@@ -8,24 +8,23 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    internal class TagHelperBinderSyntaxTreePass : IRazorSyntaxTreePass
+    internal class DefaultRazorTagHelperBinderPhase : RazorEnginePhaseBase, IRazorTagHelperBinderPhase
     {
         private static HashSet<char> InvalidNonWhitespaceNameCharacters = new HashSet<char>(new[]
```

Comment:
Yes it is. [Here](https://github.com/aspnet/Razor/blob/ajbaaska/tgbinder-phase/src/Microsoft.AspNetCore.Razor.Language/DefaultRazorTagHelperBinderPhase.cs#L246)